extensions/dicom-p10-downloader/src/commandsModule.js

```diff
@@ -0,0 +1,85 @@
+import OHIF from '@ohif/core';
+import {
+  save,
+  getDicomWebClientFromContext,
+  getSOPInstanceReferenceFromActiveViewport,
+  getSOPInstanceReferencesFromViewports,
+} from './utils';
+import downloadAndZip from './downloadAndZip';
+
+export function getCommands(context) {
+  const actions = {
+    // Example for running this command using Commands Manager
```

Comment:
You can add examples using jsdoc using the `@example` tag, like [here](https://github.com/cornerstonejs/cornerstoneTools/blob/e38e2f5c81b0f83cca5a29a03334bee287028a8a/src/store/setToolMode.js#L23).

Reply:
Nice! I'll do that! Thank you!